pkg/cloudprovider/cloud.go

```diff
@@ -160,3 +160,10 @@ type Zones interface {
 	// GetZone returns the Zone containing the current failure zone and locality region that the program is running in
 	GetZone() (Zone, error)
 }
+
+type ServiceAnnotation map[string]string
```

Comment:
To retrieve annotation values (GetValue etc). It's more useful with >1 annotation obviously. If we move the annotation to a field, or rename it, we won't have to update the code.